about.html

```diff
@@ -0,0 +1,65 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+    <title>Elearnig Website 2022, 2023</title>
+    <link rel="stylesheet" href="./style.css" >
+    <script defer src="./script.js"></script>
+    <script src="https://kit.fontawesome.com/10729b3e67.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+  </head>
+  <body class="about-body">
+      <header>
+        <div class="top-header">
+          <figure class="humburger fa-beat">
+            <span  class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+           </figure>
+           <a href="#" id="exit" class="nullifier"><i class="fa-solid fa-xmark fa-2xl fa-beat"></i></a>
+          <ul>
+            <li><a href="#"><i class="fa-brands fa-facebook-f fa-lg"></i></a></li>
+            <li><a href="#"></a><i class="fa-brands fa-twitter fa-lg"></i></li>
+            <li>English</li>
+            <li>MyPage</li>
+            <li>Layout</li>
+          </ul>
+         </div>
+         <div class="sec-header">
+          <div class="logo">
+            <a href="./index.html"> <img src="./images/logo.png" alt="logo for e learning website" width="100" height="50"></a>
+          </div>
+          <nav class="nav">
+           <ul class="toggle">
+                <li><a href="./index.html" class="">Home</a> </li>
+                <li><a href="#" class="">Program</a> </li>
+                <li><a href="#" class="">Courses</a> </li>
+                <li><a href="#" class="start-learning" id="start-learning">Start Learning</a></li>
+             </ul>
+          </nav>
+        </div>
+      </header>
+      <main>
+        <section class="about-container">
+          <h1 class="about-title">A GLOBAL E-LEARNING <br> WEBSITE | 2022 </h1>
+          <div>
+            <p>Students have the freedom to juggle their careers and school because they aren't tied down to a fixed schedule.  </p>
+            <p>Uses include the following - added Flexibility and Self-Paced Learning, better Time Management, demonstrated Self-Motivation, improved Virtual Communication and Collaboration.</p>
+            <p>Online learning is a popular alternative to traditional on-campus, in-person learning.Online education can cost less due to a variety of reasons. For example, there is no cost for commuting.</p>
+          </div>
+        </section>
+      </main>
```

Comment:
- [ ] [OPTIONAL] It seems that the rest of the about page is messing kindly consider making sure to implement the whole page not just the header section and navbar

|Your about page|The provided design|
|-|-|
|![Screenshot_20221203_103101](https://user-images.githubusercontent.com/25227022/205434129-f57cc014-37dd-4765-b01b-335f14e02793.png)|![Screenshot_20221203_103134](https://user-images.githubusercontent.com/25227022/205434139-695a5cde-5f81-4382-8add-cc51a6aff7bf.png)|